_posts/2021-02-01-denbi-brochure.md

```diff
@@ -0,0 +1,24 @@
+---
+site: freiburg
+title: "Two articles related to COVID-19 research by the European Galaxy team in the de.NBI brochure"
+tags: [article, COVID-19]
+supporters:
+ - denbi
+---
+
+<br>
+Two articles by the European Galaxy team have been included in the [de.NBI brochure __Data analysis for the COVID-19 Research__'](https://www.denbi.de/images/Downloads/deNBI_COVID19_brochure.pdf).
```

Comment:
there is a third one included about Glassgo in Galaxy, do we want to include it here?

Reply:
as you like, I can change the focus to "Galaxy-related" instead of "written by the EU Galaxy team"

Reply:
ok, let's do it this way

Reply:
"this way" means changing the focus?

Reply:
yes